lib/connection.js

```diff
@@ -112,15 +113,26 @@ class Connection extends EventEmitter {
   }
 
   // Send a handshake packet over the connection
-  //   appName - name of an application to connect to
+  //   app - string or object, application to connect to as 'name' or
+  //         'name@version' or { name, version }
```

Comment:
Do we really need all the three of these? No strong opinion on it, but I'd rather make `name@version` the only format.